lang/src/org/partiql/lang/planner/transforms/AstToLogicalVisitorTransform.kt

```diff
@@ -67,6 +69,18 @@ internal class AstToLogicalVisitorTransform(
         }
     }
 
+    // This transformation is used for top-level expression transformations and for the SFW clauses prior to the
+    //  `aggregation` relational algebra operator.
+    override fun transformExprCallAgg(node: PartiqlAst.Expr.CallAgg): PartiqlLogical.Expr = PartiqlLogical.build {
+        call(
+            "${CollectionAggregationFunction.collectionAggregationPrefix}${node.funcName.text}",
```

Comment:
Fine for now, but assume "builtins" is not treated specially. That is, in a world where functions are pluggable, we shouldn't have compiler logic dependent on a particular function library.